birdhouse/README.rst

```diff
@@ -257,6 +257,55 @@ to ``bump2version`` to auto-generated the tag in other to apply it directly on `
 after the merge, instead of onto the commits in the PR prior merging.
 
 
+Release Procedure
+-----------------
+
+* Pull/merge latest ``master`` to make sure modifications applied in
```

Comment:
missing "[are] applied in ..."

Reply:
2c7e21370c6097630b936c635e5458f796a7f559